README.md

```diff
@@ -50,6 +50,20 @@ manages the following kind of services:
   Alternatively, any other application can be specified as a source of
   replication.
 
+![Composing Microservices](assets/composing-microservices.png "Composing Microservices")
```

Comment:
We should keep the original image from the paper because it matches to the screenshot above.

Please, also try to use it as SVG because the other images are also in SVG and then with to the page width.